site/static/admin/config.yml

```diff
@@ -15,17 +15,27 @@ public_folder: /assets
 collections:
   - name: articles
     label: Articles
-    label_singular: 'Article'
+    label_singular: "Article"
     folder: content/articles
     create: true
     fields:
-      - { label: "Call Centre Only",
+      - {
+          label: "Call Centre Only",
           name: "cconly",
           widget: boolean,
           default: false,
           required: false,
-          hint: "Remember to add *CC only* at the beginning of the title. The article will be visible only to the Call Centre." }
-      - { label: "Draft Reason",
+          hint: "Remember to add *CC only* at the beginning of the title. The article will be visible only to the Call Centre.",
+        }
+      - {
+          label: "CFS Notes",
+          name: "cconlynote",
+          widget: "string",
+          hint: "This field is only visible on https://cc.census.selfhelp.onsdigital.uk/",
```

Comment:
Live domain in public repo

Reply:
changed - it went in because it was copy pasted as what Angeliki wanted it to say